composer.json

```diff
@@ -2,7 +2,7 @@
     "name": "php-pm/php-pm",
     "license": "MIT",
     "require": {
-        "php": ">=5.6.0",
+        "php": ">=7.0",
```

Comment:
Why don't you use ^7.0 instead?

Reply:
That would work as well.

Reply:
Yes it will. But ^7.0 says you are compatible with all 7.x versions. >=7.0 says you are compatible with ALL versions above 7.0 (7.0 included)